designs/l1_l4_security/l1_l4_security.md

```diff
@@ -0,0 +1,75 @@
+# L1-L4 security leveraging OpenStack security groups
+This document describes the notifications available for OpenStack security groups
+
+## Assignees
+Intel's OSC core team
+
+## Background
+OSC will support single page view to manage OpenStack security groups. User will be able to manage the OpenStack security groups and security group rules from OSC.
+
+## Constraints and Assumptions
+No addtional security groups are configured by user or admin directly on OpenStack.
+
+## Notifications provided by OpenStack
```

Comment:
The scope of the US is to research about the notifications generated for create/update/delete security group and create/delete security group rule. Also, we have mentioned about what OSC is expected to do when it receives notifications on RabbitMQ. [More info](https://github.com/larkinscarvalho/community/blob/98c1ec91579a89be3244ed5f86005ea67b7102f5/designs/l1_l4_security/l1_l4_security.md#notifications-provided-by-openstack).